src/components/QuizSetup.tsx

```diff
@@ -46,8 +39,11 @@ const QuizSetup = ({ setAllQuestions, setQuizStatus }: QuizSetupProps) => {
     }
   };
 
+  const isBeginQuizBtnDisabled =
+    selectedCategoryInfo.name === "" ? true : false;
```

Comment:
`selectedCategoryInfo.name === ""` will return a boolean, so we can simplify to this:

```suggestion
  const isBeginQuizBtnDisabled = selectedCategoryInfo.name === ""
```

Reply:
Done